### PR TITLE
[WIP] Switching from CSS to SCSS

### PR DESCRIPTION
**Pull request by shaundillon**
Because maintaining CSS is a faff.

To do:
- [x] Clean up anything the reset mucked up
- [x] Define and parameterise colours
- [x] "" spacing
- [x] "" font sizes
- [ ] Refactor the separate controller/directive CSS into something more generic
- [ ] Get the runner scripts to watch for changes and recompile

### CODE REVIEW

File: kahuna/public/config.js [hunk 1]
```diff
@@ -63,6 +64,12 @@ System.config({
     "github:jspm/nodelibs-buffer@0.1.0": {
       "buffer": "npm:buffer@3.5.3"
     },
+    "github:jspm/nodelibs-constants@0.1.0": {
```
Comment (ShaunYearStrong) [inline kahuna/public/config.js]: I have no idea where most of this came from.

File: kahuna/public/stylesheets/modules/_typography.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/** =============================================
+
+_TYPOGRAPHY.SCSS - Type styles and sizes
+
+============================================= **/
+
+$baseFontFamily: "Guardian Agate Sans Web", Georgia;
```
Comment (paperboyo) [inline kahuna/public/stylesheets/modules/_typography.scss]: Georgie is a serif font. Better fallbacks for Agate would be
`"Lucida Sans Unicode", "Lucida Grande", sans-serif`
I think.

**Comment by paperboyo** [comment]
Cool! We could, maybe, use this opportunity to introduce proper italics fonts. It will add extra weight to the app (hopefully, just once, as they should be cached), but should prevent ugliness like e.g.:

picture 38

(they should be full charset, too - "original")

File: kahuna/public/stylesheets/modules/_typography.scss (hunk 2 above)
Comment (ShaunYearStrong) [inline kahuna/public/stylesheets/modules/_typography.scss]: Cool, noted!

**Comment by paperboyo** [comment]
Fixes: https://github.com/guardian/grid/issues/280. Or it would.

**Comment by shaundillon** [comment]
Closing due to inactivity

**Pull request closed without merging**